plugins/woocommerce/src/Internal/TransientFiles/TransientFilesEngine.php

```diff
@@ -51,12 +48,12 @@ class TransientFilesEngine implements RegisterHooksInterface {
 	 * Register hooks.
 	 */
 	public function register() {
-		self::add_action( self::CLEANUP_ACTION_NAME, array( $this, 'handle_expired_files_cleanup_action' ) );
-		self::add_filter( 'woocommerce_debug_tools', array( $this, 'add_debug_tools_entries' ), 999, 1 );
+		add_action( self::CLEANUP_ACTION_NAME, array( $this, 'handle_expired_files_cleanup_action' ) );
+		add_filter( 'woocommerce_debug_tools', array( $this, 'add_debug_tools_entries' ), 999, 1 );
 
-		self::add_action( 'init', array( $this, 'add_endpoint' ), 0 );
-		self::add_filter( 'query_vars', array( $this, 'handle_query_vars' ), 0 );
-		self::add_action( 'parse_request', array( $this, 'handle_parse_request' ), 0 );
+		add_action( 'init', array( $this, 'add_endpoint' ), 0 );
```

Comment:
Even though this method was already public, should we add the `@internal` prop to it?